### ISSUE TITLE
 Html version should *not* display citation keys! 

### ISSUE THREAD

**Issue by canonicalform** [arXiv/html_feedback#2078]
### Description

Please note that unlike the pdf which shows some standard abbreviation for citations this html generator is simply showing the source code citation key. Thats a terrible idea because people use all kinds of shorthands and also stitch together different bibliographies or borrow from Google scholar and don't always uniformise the style because they know its not visible in the pdf. I'm mortified to see "Mellit-and-co". I couldn't remember his junior authors names but don't wish to cut them out!! 

### (Required) What is this issue most closely related to? Select one.

References

### Internal issue ID

cdafa0ec-1857-4605-9c02-970308bf7563

### Paper URL

https://arxiv.org/html/2409.05613v1

### Browser

Chrome/128.0.0.0

### Device Type

Android

**Comment by dginev** [comment]
@canonicalform note that while improving biblatex support here is a completely valid request (and thank you for reporting that), the *full sources* of your arXiv submissions are public.

This means that anyone who downloads the sources can inspect the citation keys used already. So some caution is always warranted.

**Comment by VladimirAlexiev** [comment]
@dginev I think the report is about inconsistency of the citation keys:
- html: `[GJV] – see [Morton-Samuelson], [MS-Daha-skein], and [Mellit-and-co]`, taken from bib source
- pdf: `[GJV23] – see [MS17], [MS21], and [Bit+21]`: formed according to particular bib style rules

Also, the "References" section is completely missing in the HTML of this paper and consequently click on a key doesn't work.

**Comment by dginev** [comment]
@VladimirAlexiev yes, I had checked the article before commenting, maybe my comment was too brief. To add more detail:

Notice also the `failed: biblatex` message in the header, which lets us know which packages weren't available during LaTeXML conversion.

The _cause_ of what you are describing is missing biblatex emulation for LaTeXML. There is already some experimental biblatex support for ar5iv, but it doesn't seem to be getting picked up during this conversion.

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.

Feel free to open new tickets for any remaining issues.
